app/static/css/detail_contract.css

```diff
@@ -0,0 +1,36 @@
+.inline {
+    display:inline;
+}
+.indent {
+    text-indent: 100px;
+}
+.indent-events {
+    text-indent: 150px;
+}
+.indent-form {
+    margin-left: 100px;
+}
+.margin-button-edit {
+    margin-top: 10px;
+}
+.margin-link {
+    margin-top: 15px;
+}
+.margin-delete {
+    margin-top: 4px;
+    margin-left: 15px;
+}
+table td {
```

Comment:
Take care of this kind of selector. You are affecting ALL the `tables td` 😱 . A better approach could create a class for these elements or at least create a class for the tables that you wants to modify.